cookbooks/aws-parallelcluster-install/recipes/nvidia.rb

```diff
@@ -83,6 +83,30 @@
     creates "/usr/local/cuda-#{node['cluster']['nvidia']['cuda_version']}"
   end
 
+  # Get CUDA Sample Files
+  cuda_tmp_samplefile = "/tmp/cuda-sample.tar.gz"
+  remote_file cuda_tmp_samplefile do
+    source node['cluster']['nvidia']['sample_url']
+    mode '0644'
+    retries 3
+    retry_delay 5
+    not_if { ::File.exist?("/usr/local/cuda-#{node['cluster']['nvidia']['cuda_version']}/samples") }
+  end
+
+  # Unpack CUDA Samples
+  bash 'cuda.sample install' do
+    user 'root'
+    group 'root'
+    cwd '/tmp'
+    code <<-CUDA
+      set -e
+      tar xf #{cuda_tmp_samplefile}
+      mv "./cuda-samples-#{node['cluster']['nvidia']['cuda_sample_version']}" "/usr/local/"
```

Comment:
Minor: could be done in one line `tar xf #{cuda_tmp_samplefile} --directory "/usr/local/"`?

Reply:
Fixed